Assets/MRTK/StandardAssets/EditorUtilities/OnLoadUtilities.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    [InitializeOnLoad]
+    public static class OnLoadUtilities
+    {
+        private const string ShaderSentinelFile = "MRTK.Shaders.Sentinel";
+        private const string ShaderImportDestination = "MRTK/Shaders";
+
+        static OnLoadUtilities()
```

Comment:
Can this follow a pattern like the one introduced in https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8286, so it doesn't try to run every time the assemblies are reinitialized? This feels like a process that only needs to happen once, and then it should be in a good state.

Reply:
We could also do it when this menu item is invoked? https://github.com/microsoft/MixedRealityToolkit-Unity/blob/06a06778e38da622b37cc299a93f16e143b7bdeb/Assets/MRTK/Core/Inspectors/MixedRealityStandardShaderGUI.cs#L781

Reply:
Also, here is the relevant check to see if the conversion is required: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/06a06778e38da622b37cc299a93f16e143b7bdeb/Assets/MRTK/Core/Inspectors/MixedRealityStandardShaderGUI.cs#L820

Reply:
Thanks for pointing that out. Missed it :)

Reply:
> We could also do it when this menu item is invoked?

It shouldnt actually be needed. The copy happens automatically.

Reply:
in fact, we _have_ to do this on import because the shader is otherwise unavailable to Unity (being in a folder called "Shaders~")